modules/hcp-bm-hosts.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disconnected/hcp-deploy-dc-bm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-bm-hosts_{context}"]
+= Creating bare metal hosts for the hosted cluster
+
+A _bare metal host_ is an `openshift-machine-api` object that encompasses physical and logical details so that it can be identified by a Metal3 Operator. Those details are associated with other Assisted Service objects, known as _agents_.
+
+.Prerequisites
+
+Before you create the bare metal host and destination nodes, you must create the virtual machines.
```

Comment:
s/you must create the virtual machines/you must have ready the destination machines, being virtual, bare metal or any other you wanna deploy on/

Reply:
Thanks. I updated the wording per your suggestion.